fe/fe-core/src/main/java/org/apache/doris/analysis/OutFileClause.java

```diff
@@ -118,12 +134,21 @@ public class OutFileClause {
     // If set to true, the brokerDesc must be null.
     private boolean isLocalOutput = false;
     private String successFileName = "";
-    private List<List<String>> schema = new ArrayList<>();
-    private Map<String, String> fileProperties = new HashMap<>();
+
+    private List<TParquetRepetitionType> schemasRepetitionType = new ArrayList<>();
+    private List<TParquetDataType> schemasDataType = new ArrayList<>();
+    private List<String> schemasColumnName = new ArrayList<>();
```

Comment:
schemasRepetitionType   schemasDataType   schemasColumnName  should be set in a struct, or else maybe cause some bug if the index is error.